src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1468,6 +1468,21 @@ private void ExecuteRestartSearchCommand(object sender, ExecutedRoutedEventArgs
                 .PostOnFailure(nameof(PackageManagerControl), nameof(ExecuteRestartSearchCommand));
         }
 
+        private void ExecuteMakeSearchCommand(object sender, ExecutedRoutedEventArgs e)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            var alternatePackageId = e.Parameter as string;
+            if (!string.IsNullOrWhiteSpace(alternatePackageId))
+            {
+                if (_windowSearchHost.IsEnabled)
+                {
+                    _windowSearchHost.SearchTask?.Stop();
+                    ActiveFilter = ItemFilter.All;
```

Comment:
Although this looks good and makes sense, can you consider not using it? 😆
We only have `ActiveFilter` for APEX, I believe. The rest of the code doesn't use it and... I think we should deprecate the Setter or entire property at some point.

```suggestion
                    _topPanel.SelectFilter(ItemFilter.All);
```

Reply:
Fixed